src/module/item/entity.js

```diff
@@ -138,32 +138,40 @@ export default class OseItem extends Item {
   }
 
   async rollFormula(options = {}) {
-    const data = this.system;
+    const itemData = this.system;
 
-    if (!data.roll) {
+    if (!itemData.roll) {
       throw new Error("This Item does not have a formula to roll!");
     }
 
     const label = `${this.name}`;
-    const rollParts = [data.roll];
+    const rollParts = [itemData.roll];
 
-    const type = data.rollType;
+    const type = itemData.rollType;
 
-    const newData = {
+    const rollData = {
       actor: this.actor,
       item: this._source,
+      description: null,
+      save: itemData.save,
+      properties: this.system.autoTags,
       roll: {
         type,
-        target: data.rollTarget,
-        blindroll: data.blindroll,
+        target: itemData.rollTarget,
+        blindroll: itemData.blindroll,
+        save: itemData.save,
```

Comment:
Is there a reason to have both `rollData.save` and `rollData.roll.save`?

Reply:
Good catch, roll.save is not necessary and was erroneously included. I've removed it in [a63e2d6](https://github.com/vttred/ose/pull/497/commits/a63e2d681c06c758257605ad2ea2c85b63d26b72)